prisma/migrations/20230701030641_add_time_to_complete_to_model_output/migration.sql

```diff
@@ -0,0 +1,8 @@
+/*
+  Warnings:
+
+  - Added the required column `timeToComplete` to the `ModelOutput` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+ALTER TABLE "ModelOutput" ADD COLUMN     "timeToComplete" INTEGER NOT NULL;
```

Comment:
Pretty sure this migration will fail in prod without a default value?

Reply:
`timeToComplete` should probably be optional anyway since if a request fails it won't be set.

Reply:
Good point, I added a default of 0